free-courses-en.md

```diff
@@ -200,6 +200,7 @@
 
 ### Deep Learning
 
+* [Complete Deep Learning processing](https://www.youtube.com/watch?v=9jA0KjS7V_c&list=PLZoTAELRMXVPGU70ZGsckrMdr0FteeRUi) - Krish Naik - (Krish Naik)
```

Comment:
Please only use the title from the source, and prefer URLs from the playlist endpoint over the watch endpoint.
Also, there's no need to specify the author twice!

```suggestion
* [Complete Deep Learning](https://www.youtube.com/playlist?list=PLZoTAELRMXVPGU70ZGsckrMdr0FteeRUi) - Krish Naik
```